### PR TITLE
Feature/model/version history

### PR DESCRIPTION
**Pull request by iroper-tessella**
Added a --version-history option to get model <version-id>.
Stores the version history in a VersionHistory object and prints it to the terminal.
Tests for the class and the get command are added.
Documentation is updated along with example outputs.

### CODE REVIEW

File: dafni_cli/model/version_history.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+            )
+            click.echo("Version message: " + version.version_message)
+            tags_string = ""
+            for tag in version.version_tags:
+                tags_string += tag + ", "
+            click.echo("Version tags: " + tags_string[:-2])
+            click.echo("")
+
+
+if __name__ == "__main__":
```
Comment (jamie-short) [inline dafni_cli/model/version_history.py:62]: assuming this isn't meant to be here?

File: dafni_cli/model/version_history.py [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+                + version.display_name
+                + TAB_SPACE
+                + "ID: "
+                + version.version_id
+                + TAB_SPACE
+                + "Date: "
+                + version.publication_time.strftime("%B %d %Y")
+            )
+            click.echo("Version message: " + version.version_message)
+            tags_string = ""
```
Comment (jamie-short) [inline dafni_cli/model/version_history.py:55]: you can simply do `tags_string = ", ".join(version.version_tags)`

`tags = ['tag_1','tag_2','tag_3']`
`", ".join(tags)`
#outputs
`'tag_1, tag_2, tag_3'`

File: test/test_get.py [hunk 3]
```diff
@@ -259,6 +263,94 @@ def test_model_methods_each_called_twice_when_two_version_ids_given(
...
+            mock_details,
+            processed_jwt_fixture,
+            get_models_list_fixture,
+            get_model_metadata_fixture,
+        ):
+            # SETUP
+            # setup get jwt
+            mock_jwt.return_value = processed_jwt_fixture, False
+            # setup retrieving
+            mock_get.side_effect = [
```
Comment (jamie-short) [inline test/test_get.py:286]: this can be `mock_get.side_effect = get_models_list_fixture`, as the fixture is an iterable

File: test/test_get.py [hunk 4]
```diff
@@ -259,6 +263,94 @@ def test_model_methods_each_called_twice_when_two_version_ids_given(
...
```
Comment (jamie-short) [inline test/test_get.py:328]: same as above

**Comment by jamie-short** [comment]
Add an `__init__.py` file into the model folder

**Review by jamie-short — CHANGES_REQUESTED** [review]
Other than a few bits of development code left in files, looks good

File: dafni_cli/model/version_history.py (hunk 1 above)
Comment (iroper-tessella) [reply dafni_cli/model/version_history.py:62]: Removed

File: dafni_cli/model/version_history.py (hunk 2 above)
Comment (iroper-tessella) [reply dafni_cli/model/version_history.py:55]: Cool, thanks! Changed.

File: test/test_get.py (hunk 3 above)
Comment (iroper-tessella) [reply test/test_get.py:286]: Good spot. Changed.

**Review by jamie-short — APPROVED** [review]
Updates look good

**Pull request merged**